language/Turkish/strings.po

```diff
@@ -843,27 +855,27 @@ msgstr "yazıcı"
 #: src/frontend/mame/ui/info.cpp:55
 msgctxt "emulation-feature"
 msgid "magnetic tape"
-msgstr ""
+msgstr "manyetik kaset"
```

Comment:
Is this translation appropriate for all forms of magnetic tape?  For example there were open reel tape formats (e.g. IBM [9-track tape](https://en.wikipedia.org/wiki/9-track_tape)) as well as cassette formats.

Reply:
as you can guess, both "magnetic" and "tape"  doesn't have a real Turkish translation. We mostly call the tape(cassette) player as "teyp" and the "tape" itself is called "kaset" which also comes from "cassette". So it shouldn't be a problem.